go/billing/models.py

```diff
@@ -120,6 +120,24 @@ def calculate_session_credit_cost(cls, session_cost, markup_percent,
             session_cost, markup_percent, context=context)
 
     @classmethod
+    def calculate_session_unit_credit_cost(
+            cls, session_length_cost, markup_percent, context=None):
+        """
+        Return the cost per session length (in credits).
+        """
+        return cls.apply_markup_and_convert_to_credits(
+            session_length_cost, markup_percent, context=context)
```

Comment:
I might be misunderstanding, but should the param maybe be named `session_unit_cost` rather than `session_length_cost`?